### PR TITLE
[dist] Correct ref to JSON path (fixes GH-4307)

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

File: src/sentry/utils/distutils/commands/build_assets.py [hunk 1]
```diff
@@ -63,14 +63,15 @@ def _get_package_version(self):
             sys.path.pop(0)
 
         if not (version and build):
+            json_path = self.get_asset_json_path()
             try:
-                with open(self.get_asset_json_path()) as fp:
-                    data = json.loads(fp.read())
+                with open() as fp:
```
Comment (mattrobenolt) [inline src/sentry/utils/distutils/commands/build_assets.py]: Uhhhh.

**Review by mattrobenolt — CHANGES_REQUESTED** [review]

File: src/sentry/utils/distutils/commands/build_assets.py (hunk 1 above)
Comment (dcramer) [inline src/sentry/utils/distutils/commands/build_assets.py]: doh looks like i hadnt hit save

Comment (mattrobenolt) [inline src/sentry/utils/distutils/commands/build_assets.py]: lol

File: src/sentry/utils/distutils/commands/build_assets.py [hunk 2]
```diff
@@ -63,14 +63,15 @@ def _get_package_version(self):
             sys.path.pop(0)
 
         if not (version and build):
+            json_path = self.get_asset_json_path()
             try:
-                with open(self.get_asset_json_path()) as fp:
-                    data = json.loads(fp.read())
+                with open(json_path) as fp:
+                    data = json.loads(fp.read(json_path))
```
Comment (mattrobenolt) [inline src/sentry/utils/distutils/commands/build_assets.py]: This is still a `TypeError` here.

**Review by mattrobenolt — APPROVED** [review]
:fire:

File: src/sentry/utils/distutils/commands/build_assets.py (hunk 2 above)
Comment (mattrobenolt) [inline src/sentry/utils/distutils/commands/build_assets.py]: dang, didn't mean to submit this since you already fixed.

**Pull request merged**